src/sourmash/minhash.py

```diff
@@ -775,3 +791,20 @@ def __setstate__(self, tup):
     def __copy__(self):
         return self
     copy = __copy__
+
+    def inflate(self, from_mh):    
```

Comment:
I think this entire method is unnecessary, since `FrozenMinHash` inherits from `MinHash`.

Reply:
So then should I just remove it completely? Or just return some particular value from it?

Reply:
please review how class inheritance works and then make a decision.